tools/accuracy_checker/accuracy_checker/preprocessor/crop.py

```diff
@@ -415,7 +494,7 @@ def process(self, image, annotation_meta=None):
 class CropOrPad(Preprocessor):
     __provider__ = 'crop_or_pad'
 
-    @classmethod
+    @ classmethod
```

Comment:
why do you add white spaces here???

Reply:
Don't know how it appeared here, deleted it.